webhook/workspace/handler/kubernetes.go

```diff
@@ -42,7 +42,8 @@ func (h *WebhookHandler) validateKubernetesObjectPermissionsOnCreate(ctx context
 			continue
 		}
 		if component.Uri != "" {
-			return fmt.Errorf("kubenetes components specified via URI are unsupported")
+			// We're going to ignore URI components for now
```

Comment:
A future improvement could be to return a webhook warning (so that the dashboard could display it) but for now, I don't think this is necessary.